lib/ansible/module_utils/redfish_utils.py

```diff
@@ -135,7 +135,7 @@ def _find_systems_resource(self, uri):
             data = response['data']
             for member in data[u'Members']:
                 systems_service = member[u'@odata.id']
-            self.systems_uri = systems_service
+                self.systems_uri = systems_service
             return {'ret': True}
```

Comment:
I think the "return" line should also be indented. After all, we need to find only one resource, not all of them.

Reply:
@dtantsur thank you for the review :)

Reply:
> I think the "return" line should also be indented. After all, we need to find only one resource, not all of them.
 
Perhaps, though this would change the outcome of this loop i.e. we would store in `self.systems_uri` first occurrence rather than the last one.

Also, in case of no members we would return `None` then.

Reply:
Thank you @etingof , I'll look into it :)